Makefile

```diff
@@ -68,27 +68,27 @@ update: | update-common
 		$(MAKE) waku.nims $(HANDLE_OUTPUT)
 
 # a phony target, because teaching `make` how to do conditional recompilation of Nim projects is too complicated
-wakunode: | build deps
+node: | build deps
```

Comment:
wakunode (and version 2) is perhaps the only one I would keep as is, as the application has the same name, but no big preference really.

Reply:
I would be more happy with changing the wakunode application name to node. Seems more logical, if I am in the waku repository I of course expect the node to be waku.

Reply:
It is a good point though, because wakunode is expected to be put in your PATH. E.g. see Dockerfile. You don't want to confuse wakunode with NodeJS ;)

All the rest are only used in the repository so can be more local, afaik.

Reply:
I'd also call this wakunode1, wakunode should be symlinked to the latest stable release (e.g)